content/influxdb3/enterprise/get-started/multi-server.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Use a multi-server setup
+seotitle: Use a multi-server InfluxDB 3 Enterprise setup
+menu:
+  influxdb3_enterprise:
+    name: Multi-server
+    parent: Get started
+weight: 4
+influxdb3/enterprise/tags: [cluster, multi-node, multi-server]
+---
+
```

Comment:
I debated including this in the get-started guides, but opted to keep it under install. Right now, the get started guides largely rely on the installation doc to cover installation. That's why I put the multi-server guide there. If we're going to keep this here, we either need to remove the install one or add `canonical` frontmatter on this one to point to the other one.